### PR TITLE
Load schema as resource

### PR DESCRIPTION
**Pull request by br648**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Loading the JSON schema as a file fails once deployed. This PR addresses that by loading the JSON schema as a resource instead.

### CODE REVIEW

File: src/main/java/org/opentripplanner/middleware/utils/ConfigUtils.java [hunk 1]
```diff
@@ -25,14 +25,14 @@
...
-    public static final String DEFAULT_ENV_SCHEMA = "src/main/resources/env.schema.json";
+    public static final String DEFAULT_ENV_SCHEMA = "env.schema.json";
     private static JsonNode ENV_SCHEMA = null;
     private static final String JAR_PREFIX = "otp-middleware-";
 
     static {
         // Load in env.yml schema file statically so that it is available for populating properties when running CI.
         try {
-            ENV_SCHEMA = yamlMapper.readTree(new FileInputStream(DEFAULT_ENV_SCHEMA));
+            ENV_SCHEMA = yamlMapper.readTree(ConfigUtils.class.getClassLoader().getResourceAsStream(DEFAULT_ENV_SCHEMA));
```
Comment (landonreed) [inline src/main/java/org/opentripplanner/middleware/utils/ConfigUtils.java]: I think this load/read stuff should go into its own method (it's used here and in the Updater class) and the javadoc should state that it must be loaded as a resource. Otherwise, we lose any context as to why this change was made and run the risk of someone changing it back to a `new FileInputStream` because it looks nicer.

**Review by landonreed — CHANGES_REQUESTED** [review]
See comment about adding a method with comments about why this approach is being used.

File: src/main/java/org/opentripplanner/middleware/utils/ConfigUtils.java (hunk 1 above)
Comment (br648) [reply src/main/java/org/opentripplanner/middleware/utils/ConfigUtils.java]: Updated.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

**Review by landonreed — APPROVED** [review]
